po/de.po

```diff
@@ -661,6 +663,8 @@ msgid ""
 "No matching actions found for this event.\n"
 "\n"
 msgstr ""
+"--- Überspringe %s ---\n"
+"Für diesen Event wurden keine passenden Aktionen gefunden."
```

Comment:
msgid and msgst must have matching number of \n (newline) characters. It leads to errors otherwise, so the

"Für diesen Event wurden keine passenden Aktionen gefunden."
should be
"Für diesen Event wurden keine passenden Aktionen gefunden.\n"